### PR TITLE
import csi-lib-iscsi to this project 

### PR DESCRIPTION
**Pull request by humblec**
Signed-off-by: Humble Chirammal <hchiramm@redhat.com>

**What type of PR is this?**
> /kind feature

**What this PR does / why we need it**:

In our previous discussions, we have decided to archive csi-lib-iscsi project and use it directly from here. 

**Special notes for your reviewer**:

This PR  import   csi-lib-iscsi to `pkg/lib/iscsi` dir 
-->
```release-note
NONE
```

### CODE REVIEW

**Comment by humblec** [comment]
Thought of preserving the history, but looking at the CLA complaints may be I can squash and bring those.

File: pkg/lib/iscsi/.github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+<!--  Thanks for sending a pull request!  Here are some tips for you:
```
Comment (andyzhangx) [inline pkg/lib/iscsi/.github/PULL_REQUEST_TEMPLATE.md:1]: remove this file

File: pkg/lib/iscsi/CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
+# Contributing Guidelines
```
Comment (andyzhangx) [inline pkg/lib/iscsi/CONTRIBUTING.md:1]: remove this file

File: pkg/lib/iscsi/LICENSE [hunk 3]
```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```
Comment (andyzhangx) [inline pkg/lib/iscsi/LICENSE:1]: remove this file

File: pkg/lib/iscsi/go.mod [hunk 4]
```diff
@@ -0,0 +1,8 @@
+module github.com/kubernetes-csi/csi-lib-iscsi
```
Comment (andyzhangx) [inline pkg/lib/iscsi/go.mod:1]: if you are not using go sub module, you don't need to copy go.mod

File: pkg/lib/iscsi/.github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (humblec) [reply pkg/lib/iscsi/.github/PULL_REQUEST_TEMPLATE.md]: done

File: pkg/lib/iscsi/CONTRIBUTING.md (hunk 2 above)
Comment (humblec) [reply pkg/lib/iscsi/CONTRIBUTING.md]: Done.

File: pkg/lib/iscsi/LICENSE (hunk 3 above)
Comment (humblec) [reply pkg/lib/iscsi/LICENSE]: Done.

**Comment by humblec** [comment]
Thanks @andyzhangx , ptal.. comments are addressed.

**Review by andyzhangx — APPROVED** [review]
/lgtm
/approve

**Pull request merged**